mindmeld/cli.py

```diff
@@ -85,6 +106,153 @@ def _app_cli(ctx):
         ctx.obj = {}
 
 
+def _dvc_add_helper(filepath):
+    """
+    Returns True if successful, False otherwise along with helper message
+
+    Args:
+        filepath (str): path to file/folder to add to DVC
+
+    Returns:
+        (tuple) True if no errors, False + error string otherwise
+    """
+    p = subprocess.Popen(
+        ["dvc", "add", filepath], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )
+    # Get DVC error message from standard error
+    _, error = p.communicate()
+    error_string = error.decode("utf-8")
+
+    if DVC_INIT_ERROR_MESSAGE in error_string:
+        return False, DVC_INIT_HELP
+    elif DVC_ADD_DOES_NOT_EXIST_MESSAGE in error_string:
+        return False, DVC_ADD_DOES_NOT_EXIST_HELP.format(dvc_add_path=filepath)
+    elif error_string:
+        return False, error_string
+    else:
+        return True, None
+
+
+def _bash_helper(command_list):
+    """
+    Helper for running bash using subprocess and error handling
+
+    Args:
+        command_list (list): Bash command formatted as a list, no spaces in each element
+
+    Returns:
+        (tuple) True if no errors, False + error string otherwise
+    """
+    p = subprocess.Popen(command_list, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    _, error = p.communicate()
+    error_string = error.decode("utf-8")
+
+    if error_string:
+        return False, error_string
+
+    return True, None
+
+
+@_app_cli.command("dvc", context_settings=CONTEXT_SETTINGS)
+@click.pass_context
+@click.option(
+    "--init", is_flag=True, required=False, help="Instantiate DVC within a repository"
+)
+@click.option(
+    "--save", is_flag=True, required=False, help="Save built models using dvc"
+)
+@click.option("--checkout", required=False, help="Instantiate DVC within a repository")
+@click.option(
+    "--help",
+    "help_",
+    is_flag=True,
+    required=False,
+    help="Print message showing available options",
+)
+def dvc(ctx, init, save, checkout, help_):
+    app = ctx.obj.get("app")
+    app_path = app.app_path
+
+    # Ensure that DVC is installed
+    if not which("dvc"):
+        logger.error(
+            "DVC is not installed. You can install DVC by running 'pip install dvc'."
+        )
+        return
+
+    if init:
+        success, error_string = _bash_helper(["dvc", "init", "--subdir"])
+        if not success:
+            logger.error("Error during initialization: %s", error_string)
```

Comment:
Let's add a help message to tell the user to either delete the .dvc folder or run dvc destroy before running init again